service-data-access/src/main/java/gov/va/vro/abd_data_access/service/FhirClient.java

```diff
@@ -71,8 +71,7 @@ public String getUrl() {
               }),
           new AbstractMap.SimpleEntry<String, AbdDomain[]>(
               "6602", new AbdDomain[] {
-                      AbdDomain.MEDICATION,
-                      AbdDomain.CONDITION
+                  AbdDomain.MEDICATION
```

Comment:
I thought we just added this CONDITION. You don't want it now?

Reply:
For version 1 I realized it is not needed